app/services/quarterly_message_employee_matcher.rb

```diff
@@ -0,0 +1,52 @@
+require "business_time"
+
+class QuarterlyMessageEmployeeMatcher
```

Comment:
yes always a tricky balance to make PRs small and clean up things as you go along! if you think of a good class name change, by all means go for it. If not then no worries.